libexec/cli/image.create.info

```diff
@@ -11,11 +11,14 @@ CREATE OPTIONS:
     -s|--size   Specify a size for an operation in MiB, i.e. 1024*1024B
                 (default 768MiB)
     -F|--force  Overwrite an image file if it exists
+    -i          Specify a size for the inode ratio in Bytes. Default is
```

Comment:
Oh sorry one more thing.  I think it's OK to have a long option without a short option, but I don't think we should have a short option without a long option.